index.html

```diff
@@ -4,10 +4,133 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="style.css">
-    <title>Document</title>
+    <link rel="stylesheet" href="index.css">
+        <script src="https://kit.fontawesome.com/187f763109.js" crossorigin="anonymous"></script>
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">          
+    <link href='https://fonts.googleapis.com/css?family=Lato' rel='stylesheet'>
+    <link rel="stylesheet" href="https://use.typekit.net/ntt5mmb.css">
+    <title>Poem catcher - A soulful Journey of a Lost Poet  </title>
+    <link rel="shortcut icon" type="img/png" href="./src/poetry-title-img.png">
 </head>
 <body>
+    <div class="landingpage-wrap">
+         <nav class=" nav-bar">
+    <nav class="top-nav">
+        <ul>
+            <li><a href="#" target="_blank"><i class="fa-brands fa-facebook-f"></i></a></li>
+            <li><a href="#" target="_blank"><i class="fa-brands fa-twitter"></i></a></li>
+            <li><a href="#">English</a></li>
+            <li><a href="#">My page</a></li>
+            <li><a href="#">Log Out</a></li>
+        </ul>
+    </nav>
+    <nav class="bottom-nav">
+        <div class="hamburger">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
+            <div class="logo-wrap">
+             <a href="./index.html"><img class="logo" src="./src/poetry-title-img.png" alt=""></a>
+
+            </div>
+            <ul class="nav-menu">
+                <li class="nav-item"><a class = "nav-link portfolio-link" href="./about.html">About</a></li>
+                <li class="nav-item"><a class =  " nav-link about-link" href="#">Poems</a></li>
+                <li class="nav-item"><a class = "nav-link contact-link" href="#">Stories</a></li>
```

Comment:
- [ ] As the previous reviewer has pointed out, on the mobile, there is an extra space to the right of the homepage that causes horizontally scrolling. Please kindly fix that.

![image](https://user-images.githubusercontent.com/43995830/175776319-86b3f7de-15cb-4112-95e1-dde0f2910b5d.png)

Reply:
- [ ] [REQUIRED] Great job @Krishnabot 👍🏽 However, as requested by the previous reviewer, you still need to ensure this part of the mobile version(Homepage) is fixed. Kindly reduce the font sizes and justify it to the center so it looks like the [BEHANCE DESIGN](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
![Screenshot from 2022-06-25 15-44-32](https://user-images.githubusercontent.com/58692590/175779118-bde0471a-f1aa-459d-b71e-309ed5042c0f.png) | ![Screenshot from 2022-06-25 15-57-56](https://user-images.githubusercontent.com/58692590/175779172-9caeec2c-e0b8-4dd6-97a4-51dca0d8514b.png)